src/hooks/useProposals.js

```diff
@@ -277,7 +283,7 @@ function processProposal(proposal, latestBlock, account, config) {
   }
 }
 
-function processDecision(proposal) {
+function processDecision(proposal,voidedDecisions) {
```

Comment:
Your Prettier its off? I think with save it should get automatic space afte comma.

Reply:
> I found you modified the dao-list repo to save the voided and there i found the garden to test
> 
> http://localhost:3000/#/rinkeby/garden/0x7777cd7c9c6d3537244871ac8e73b3cb9710d45a
> 
> looks great! That was fast feature implementation. Good job! @rperez89

ohh sorry i should have explainded how to test it